packages/kbn-optimizer/src/integration_tests/basic_optimization.test.ts

```diff
@@ -232,7 +232,8 @@ it('uses cache on second run and exist cleanly', async () => {
   `);
 });
 
-it('prepares assets for distribution', async () => {
+// test is failing since actual snapshot is modified when running with code coverage
+(!!process.env.CODE_COVERAGE ? it.skip : it)('prepares assets for distribution', async () => {
```

Comment:
This test fails when running with `--coverage` flag since snapshot is modified with code coverage data. Skipping it.